pkg/imgpkg/bundle/images_lock.go

```diff
@@ -26,6 +26,38 @@ type ImagesLock struct {
 	relativeToRepo string
 }
 
+func (o ImagesLock) ImageRefs() []lockconfig.ImageRef {
+	return o.imagesLock.Images
+}
+
+func (o *ImagesLock) Merge(imgLock ImagesLock) error {
```

Comment:
since NewImagesLock constructor returns pointer-ed ImagesLock lets accept it here as well.